### ISSUE TITLE
Add `aria-invalid` attribute to input fields

### ISSUE THREAD

**Issue by obliviga** [department-of-veterans-affairs/va-forms-system-core#476]
All input fields within a form should have the `aria-invalid` attribute. If the field is invalid, then the `aria-invalid` attribute's value should be set to `true`. The default value for this attribute should be `false`. If the input field has an `aria-required` attribute, then the `aria-invalid` attribute should be omitted.

https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA21

**Comment by obliviga** [comment]
Turns out this feature is in progress here: https://github.com/department-of-veterans-affairs/component-library/pull/489
